cloudprober/templates/deployment.yaml

```
@@ -42,6 +42,17 @@ spec:
               {{- toYaml . | nindent 14 }}
             {{- end }}
         {{- end }}
+        {{- range .Values.extraSecretMounts }}
+        - name: {{ .name }}
+          secret:
+            secretName: {{ .secretName | default .name }}
+            defaultMode: {{ .defaultMode | default 400 }}
+            {{- with .items }}
+            items:
+
+              {{- toYaml . | nindent 14 }}
```

Drop this extra line?
```suggestion
            items:
              {{- toYaml . | nindent 14 }}
```